lib/resources/content/babelrc.js

```diff
@@ -2,26 +2,23 @@
 module.exports = () => {
   return {
     "plugins": [
+      "syntax-flow",
       "transform-decorators-legacy",
-      "transform-class-properties"
+      "transform-flow-strip-types"
     ],
     "presets": [
       [
         "env", {
           "targets": process.env.BABEL_TARGET === 'node' ? {
             "node": process.env.IN_PROTRACTOR ? '6' : 'current'
           } : {
-            "browsers": [
-              "last 2 versions",
-              "not ie <= 11"
-            ],
-            "uglify": process.env.NODE_ENV === 'production',
+            "browsers": [ "last 2 versions" ]
           },
           "loose": true,
-          "modules": process.env.BABEL_TARGET === 'node' ? 'commonjs' : false,
-          "useBuiltIns": true
+          "modules": process.env.BABEL_TARGET === 'node' ? 'commonjs' : false
         }
-      ]
+      ],
+      "stage-1"
```

Comment:
For webpack, use "stage-1" to replace handpicked "transform-class-properties".
Added "syntax-flow" for webpack setup too.